### PR TITLE
[Fix] CodeAutoFill mixin is missing unregisterListener function

### PR DESCRIPTION
**Pull request by astralstriker**

### CODE REVIEW

File: example/.flutter-plugins-dependencies [hunk 1]
```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"sms_autofill","dependencies":[]}]}
```
Comment (jaumard) [inline example/.flutter-plugins-dependencies:1]: shouldn't be in version control please delete it

File: example/ios/Flutter/flutter_export_environment.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (jaumard) [inline example/ios/Flutter/flutter_export_environment.sh:1]: shouldn't be in version control please delete it

**Comment by jaumard** [comment]
looks good but please remove extra files not needed, after that it can be merged

**Comment by astralstriker** [comment]
sure

**Comment by jaumard** [comment]
I've integrated those changes into last version :) closing

**Pull request closed without merging**